journal/modernized/33-testimonies.adoc

```diff
@@ -305,10 +300,8 @@ Joseph Pike.
 
 Waterford, the 2nd of the Ninth month, 1713.
 
-[.asterism]
-'''
-
-=== The Testimony of Friends of Mount-melick Monthly Meeting; concerning our dear and ancient friend William Edmundson, whom the Lord has been pleased to remove from us by death; and though it he our loss, we believe it is his great gain.
+=== The Testimony of Friends of Mount-melick Monthly Meeting; concerning our dear and ancient friend William Edmundson, whom the Lord has been pleased to remove from us by death; and though it be our loss, we believe it is his great gain.
+//I'm not sure if that was appropriate change above, but I can't find this whole chapter in either of the PDF's. Everything I search for in both of them doesn't come up.
```

Comment:
Here's the link: https://books.google.com/books?id=NQ9NAAAAcAAJ&pg=PR37&lpg=PR37&dq=The+Testimony+of+Friends+of+Mount-melick+Monthly+Meeting;+concerning+our+dear+and+ancient+friend+William+Edmundson,+whom+the+Lord+has+been+pleased&source=bl&ots=FkuMhaofOV&sig=ACfU3U0MdEbaPjJ1wuz5tWq4sKrpUsnXow&hl=en&sa=X&ved=2ahUKEwiyqo2O07HhAhUMP6wKHaujCy4Q6AEwAHoECAkQAQ#v=onepage&q=The%20Testimony%20of%20Friends%20of%20Mount-melick%20Monthly%20Meeting%3B%20concerning%20our%20dear%20and%20ancient%20friend%20William%20Edmundson%2C%20whom%20the%20Lord%20has%20been%20pleased&f=false

Maybe you're getting thrown off because we always move the testimonies to the end of the book, but they often appear at the beginning of the google pdfs.

Reply:
I think this is the one I was using, and searching for lines brought up nothing! It happens from time to time, and I've figured out it's because sometimes there's a change from what's in the PDF, but I tried several different lines in both editions and got nuthin! Idk! 🤷‍♀️